libshaderc_util/src/compiler.cc

```diff
@@ -243,8 +243,13 @@ std::tuple<bool, std::vector<uint32_t>, size_t> Compiler::Compile(
   // 'spirv' is an alias for the compilation_output_data. This alias is added
   // to serve as an input for the call to DissassemblyBinary.
   std::vector<uint32_t>& spirv = compilation_output_data;
+  glslang::SpvOptions options;
+  options.generateDebugInfo = false;
+  options.disableOptimizer = true;
```

Comment:
This was surprising to see, but yes, this maintains previous behaviour.
I got tripped up because there are still OpName instructions in the output, but that's Glslang's choice for now.